### ISSUE TITLE
Build failing with "Type 'TabViewStyle' has no member 'sidebarAdaptable'"

### ISSUE THREAD

**Issue by matinzd** [okwasniewski/react-native-bottom-tabs#24]
The build is failing after installing.

## Log

```
Type 'TabViewStyle' has no member 'sidebarAdaptable'
```

<img width="1385" alt="image" src="https://github.com/user-attachments/assets/29bafaae-0cc3-4e6c-969e-88d20d60bcaa">

## Environment

React Native: 0.75.4

**Comment by okwasniewski** [comment]
Hey, 

Thanks for opening the issue. This is caused by using older version of Xcode. 

Any chance you could try submitting a PR fixing this? I don't have Xcode 15.

I think this could be fixed by adding #if compiler(>=5.9)

Thanks in advance!

**Comment by matinzd** [comment]
This fixes the issue:

```swift
        #if compiler(>=6.0)
          self.tabViewStyle(.sidebarAdaptable)
        #endif
```
